packages/sources/deutsche-boerse/src/endpoint/lwba.ts

```diff
@@ -30,12 +30,22 @@ export const inputParameters = new InputParameters(
   ],
 )
 
-interface LwbaLatestPriceResponse {
+interface LwbaResponse {
   Result: number | null
   Data: {
     mid: number
     bid: number
     ask: number
+    bidSize: number
+    askSize: number
+    quoteProviderIndicatedTimeUnixMs: number
+    tradeProviderIndicatedTimeUnixMs: number
```

Comment:
No need to have this now, having those 2 timestamps here was because we had trade and quote in the same response, now we can remove both.